src/components/Pill/src/Pill.vue

```diff
@@ -14,6 +14,15 @@
 import cssValidator from '@square/maker/utils/css-validator';
 import { MThemeKey, defaultTheme, resolveThemeableProps } from '@square/maker/components/Theme';
 
+const MAXIMUM_ALLOWED_SHAPES = 2;
+const ALLOWED_SHAPES = new Set([
+	'sharp',
```

Comment:
The name `sharp` is a little unintuitive to me (also not a shape). Could `rectangle` be a better name?

<img width="880" alt="Screenshot 2023-02-02 at 12 10 54 PM" src="https://user-images.githubusercontent.com/1075694/216222198-adef02bb-eea0-4c01-a1a9-c68150f7be7d.png">

Reply:
The convention in Maker across all other components that have a `shape` prop is to use the names: `pill`, `rounded`, and `squared`. Please rename `sharp` to `squared` so it's consistent with the rest of the library. I think the custom fancy shape names being `point` and `ribbon` are fine.